django_socio_grpc/tests/fakeapp/utils.py

```diff
@@ -0,0 +1,18 @@
+import sys
+from importlib import reload
+
+from django_socio_grpc.utils.registry_singleton import RegistrySingleton
+from django_socio_grpc.utils.servicer_register import AppHandlerRegistry
+
+
+def make_grpc_handler(name, *services, reload_services=False):
+    def grpc_handler(server):
+        if reload_services:
+            RegistrySingleton().clean_all()
+        app_registry = AppHandlerRegistry(name, server)
```

Comment:
actually why not creating a method in AppHandlerRegistry to do that instead of adding a new method in a utils file ?

Reply:
Added `reload_services` argument to `AppHandlerRegistry`